### PR TITLE
docker: use ceph_docker_osd_devices variable for purging docker cluster

### LINKED ISSUE #727: docker: make purge playbook use devices variables
Use variables in `ceph_osd_docker_devices` instead of a hardcoded list

### PR DESCRIPTION
**Pull request by ivotron**
Also checks for the existence of gdisk and installs if necessary. fixes #727

### CODE REVIEW

File: purge-docker-cluster.yml [hunk 1]
```diff
@@ -8,36 +8,32 @@
 
   tasks:
 
+  - name: install the latest version of gdisk
+    package: name=gdisk state=present
```
Comment (leseb) [inline purge-docker-cluster.yml]:
```
package:
  name: disk
  state: present
```

**Comment by leseb** [comment]
@ivotron up :)

**Comment by ivotron** [comment]
Added corrections. Thanks!

File: purge-docker-cluster.yml [hunk 2]
```diff
@@ -8,36 +8,34 @@
...
+    shell: docker ps -aq --filter="ancestor=ceph/daemon"
     register: containers
 
   - name: delete ceph containers
     shell: docker rm -f {{ item }}
     with_items: containers.stdout_lines
 
   - name: purge ceph directories
-    shell: rm -rf {{ item }}
+    file: path={{ item }} state=absent
```
Comment (leseb) [inline purge-docker-cluster.yml]:
```
file:
  path:
  state:
```

:)

**Comment by alfredodeza** [comment]
test this please

**Comment by leseb** [comment]
test this please

File: purge-docker-cluster.yml (hunk 2 above)
Comment (ivotron) [inline purge-docker-cluster.yml]: missed that. Fixed on commit below. thanks!

**Pull request merged**